UI/DarwinFileSystemServices.h

```diff
@@ -1,12 +1,12 @@
 //
-//  DarwinMemoryStickManager.h
+//  DarwinFileSystemServices.h
 //  PPSSPP
 //
 //  Created by Serena on 20/01/2023.
 //
 
-#ifndef DarwinMemoryStickManager_h
-#define DarwinMemoryStickManager_h
+#ifndef DarwinFileSystemServices_h
+#define DarwinFileSystemServices_h
```

Comment:
Since you're changing this anyway, let's switch to #pragma once, not that it matters much.